lib/accessory.js

```diff
@@ -54,15 +64,19 @@ class Accessory {
   }
 
   setOutlet (uuid, powered, done) {
-    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {
+	  var that = this;
```

Comment:
As you might have picked up on, I'm more of a part-time code hacker so I wasn't really familiar with the whole => convention I saw.  I just plowed through.  What is that?  CoffeeScript or something else?  Will gladly fix on my end if I can learn along the way though.